nixos/modules/config/users-groups.nix

```diff
@@ -616,9 +622,11 @@ in {
         # there is at least one "privileged" account that has a
         # password or an SSH authorized key. Privileged accounts are
         # root and users in the wheel group.
-        assertion = !cfg.mutableUsers ->
-          any id ((mapAttrsToList (_: cfg:
-            (cfg.name == "root"
+        # The check does not apply when users.disableLoginPossibilityAssertion
```

Comment:
This seems to be an artefact of an earlier naming of the option.
This should be `users.allowNoPasswordLogin` if I'm not mistaken.